Mc speed up

aleCombi opened the pull request.

aleCombi committed:
Updated roadmap

aleCombi committed:
first draft of my EnsembleSol with reuse of noiseproblem

aleCombi committed:
first sketch of fast noise solvers

aleCombi committed:
Benchmarking speed up for montecarlo exact

aleCombi committed:
First quick solve with alloc

aleCombi committed:
Best alloc free version so far

aleCombi committed:
Working on Montecarlo using marginal law for exact simulation and euro payoffs

aleCombi committed:
New way to do montecarlo sampling from Distributions.rand

aleCombi committed:
Seeding montecarlo sampler

aleCombi committed:
Added license badge

aleCombi committed:
Documenting montecarlo

JADekker commented:
I've glanced through the diff, looks promising! I'll try to give the code a look next week. Do you have any nice benchmarking results to show the improvement over the current?

aleCombi committed:
Consolidated montecarlo european solvers

aleCombi committed:
Added docstrings

aleCombi committed:
Moved heston to log space

aleCombi committed:
Debugging heston exact

aleCombi committed:
Correcting sampling of log_price for heston

aleCombi committed:
Added heston montecarlo agreement tests and checks for seed numbers in SimulationConfig builder.

aleCombi committed:
Testing LSM against binomial tree

aleCombi committed:
Little corrections

aleCombi committed:
more euler heston paths

aleCombi commented:
@JADekker I did some work here, cleaned up the code, solved the bugs I introduced and added some tests.
I will add some performance benchmark in the examples folder.

aleCombi committed:
Added montecarlo methods benchmark

aleCombi committed:
Removed old
 working examples

aleCombi committed:
Trigger CI

aleCombi committed:
CI name change to allow PR checks succeeding
